recipes/c-blosc2/all/conanfile.py

```diff
@@ -19,7 +21,7 @@ class CBlosc2Conan(ConanFile):
         "fPIC": [True, False],
         "simd_intrinsics": [None, "sse2", "avx2"],
         "with_lz4": [True, False],
-        "with_zlib": [None, "zlib", "zlib-ng"],
+        "with_zlib": [None, "zlib", "zlib-ng", "zlib-ng-compat"],
```

Comment:
what is the real meaning of this new `zlib-ng-compat` here? Does it make sens here or is it something which should be controlled from zlib-ng options rather than c-blosc2 option?